### DISCUSSION TITLE
阅读wiki-Basic syntax of note type时发现了的一个小问题

### DISCUSSION THREAD

**Discussion by LZhaoM** [Rainbow-Dreamer/musicpy#56] [General]
在 [https://github.com/Rainbow-Dreamer/musicpy/wiki/Basic-syntax-of-note-type](url) 里， **Converting pitch numbers of notes to note types and getting pitch numbers from note types** 中第一个例子 `degree_to_note(60)` 下面我认为应该是C4而不是C5。
谢谢

**Comment by Rainbow-Dreamer** [comment]
感谢发现wiki的问题，确实应该是C4，这边会进行修改
